client/src/rpc_client.rs

```diff
@@ -55,6 +55,29 @@ impl RpcClient {
         }
     }
 
+    pub fn confirm_transaction(
```

Comment:
should this be `confirm_transaction_with_commitment`, with a corresponding `confirm_transaction` method using `CommitmentConfig::default()`? (following the `get_balance` pattern)

Reply:
Sure